dev/test-studio/schema/standard/portableText/simpleBlock.js

```diff
@@ -69,7 +75,15 @@ export default {
             spellCheck: true,
           },
         },
-        {type: 'image', name: 'image'},
+        {
+          type: 'image',
+          name: 'image',
+          options: {
+            modal: {
+              // width: 'small',
```

Comment:
remove or add a note about why it's still here / when it can be removed?

Reply:
This is Test Studio only, thought it would be good to leave it as documentation. But will add comments 👍